python/across_api/across/hello.py

```diff
@@ -0,0 +1,48 @@
+from typing import Optional
+
+from ..base.common import ACROSSAPIBase
+from .schema import HelloGetSchema, HelloSchema
+
+
+class Hello(ACROSSAPIBase):
+    """Sample ACROSS API Class
+
+    Parameters
+    ----------
+    name : str
+        Your name, if you wish to give it
```

Comment:
Is it conventional that if we are annotating types that we _also_ need to supply the types in Numydoc-style docstrings? Seems like there should be a way to avoid repeating ourselves.

Reply:
The [numpy docstring style guide](https://numpydoc.readthedocs.io/en/latest/format.html) says to indicate type if it is specified, and in strongly typed code like this I would say we should always put in type. This does place the burden of making sure code and comments are consistent of course. However, I would suggest that the current usage is the correct one.

Reply:
a8b1c4b